src/core/credrank/markovProcessGraph.test.js

```diff
@@ -183,6 +183,14 @@ describe("core/credrank/markovProcessGraph", () => {
             )
             // .2 -- because this has incident organic edges
             .set(2, parameters.alpha)
+            // .55 -- because this one has no incident organic edges
+            .set(
+              4,
+              1 -
+                parameters.gammaForward -
+                parameters.gammaBackward -
+                parameters.beta
```

Comment:
TBH, I don't get this at all. I just ran the test with a random value, and then found out the expected value from the failed test report and put it into the test. So please verify that this is in fact the expected value.